### PR TITLE
Fix #1413

### LINKED ISSUE #1413: csharp input files out of date
`Dict.i` needs to be included like it is for the Java wrappers:
https://github.com/rdkit/rdkit/blob/master/Code/JavaWrappers/gmwrapper/GraphMolJava.i#L181
`MonomerInfo.i` is also missing.

### PR DESCRIPTION
**Pull request by greglandrum**
Fixes #1413
This is a small change to allow the csharp SWIG wrappers to build

### CODE REVIEW

File: Code/JavaWrappers/FilterCatalog.i [hunk 1]
```diff
@@ -163,7 +163,7 @@
 %typemap(javaout) std::string RDKit::FilterCatalog::Serialize {
   return $jnicall;
 }
-%typemap(out) std::string RDKit::FilterCatalog::Serialize {
+%typemap(javaout) std::string RDKit::FilterCatalog::Serialize {
```
Comment (bp-kelley) [inline Code/JavaWrappers/FilterCatalog.i]: This appears to break the java-builds, although it does look correct...

Comment (bp-kelley) [inline Code/JavaWrappers/FilterCatalog.i]: This should indeed be %typemap(out), it needs to return a jbytearray and not a javastring

**Comment by greglandrum** [comment]
It needs to be built for the java wrappers only.
`NewByteArray` seems to be a Java thing, so the csharp build doesn't work with `typemap(out)`

**Comment by greglandrum** [comment]
@bp-kelley : that commit allows both the csharp and java wrappers to build.
Thanks for pointing out the problem.

**Pull request merged**